designsafe/static/scripts/workspace/html/data-browser.html

```diff
@@ -63,28 +63,49 @@ <h2>Data Depot Browser</h2>
                         <td ng-if="f.meta">{{f.meta.title}}</td>
                         <td ng-if="f.metadata">{{f.metadata.project.title}}</td>
                     </tr>
-                <tr ng-if="data.filesListing.system !== 'nees.public'" ng-repeat="f in data.filesListing.children | orderBy : ['type', 'name']">
-                    <td>
-                        <button ng-if="f.name !== '.' && data.wants"
-                                type="button"
-                                class="btn btn-xs btn-info"
-                                ng-click="chooseFile(f)">Select</button>
-                        <a ng-click="browseFile(f)" class="btn btn-xs btn-link" ds-data-draggable data-ds-data="agave://{{f.system}}/{{f.path}}">
-                            <i class="fa {{f.icon()}}"></i>
-                            <span>{{renderName(f)}}</span>
-                        </a>
-                    </td>
-                    <td>{{f.length|bytes:0}}</td>
-                </tr>
-                <tr ng-if="data.loading"><td colspan="3"><p class="lead"><i class="fa fa-spin fa-circle-o-notch"></i> Loading files...</p></td></tr>
-                <tr ng-if="data.error"><td colspan="3"><p class="alert alert-danger">{{data.error}}</p></td></tr>
-                <tr ng-if="data.busyListingPage">
-                  <td>
-                        <span class="bg-info text-info">
-                           <i class="fa fa-spinner fa-spin"></i> Loading...
-                        </span>
-                  </td>
-                </tr>
+                    <tr ng-if="data.filesListing.system !== 'nees.public' && data.filesListing.system === 'designsafe.storage.projects'" 
```

Comment:
Can get rid of the `nees.public` term here
```suggestion
                    <tr ng-if="data.filesListing.system === 'designsafe.storage.projects'" 
```

Reply:
Done